src/hist/__init__.py

```diff
@@ -5,13 +5,12 @@
 
 from types import ModuleType
 
-from . import accumulators, axis, numpy, storage, tag
+from . import accumulators, axis, intervals, numpy, storage, tag
```

Comment:
This forces `intervals` to be importable, which in turn forces SciPy to be importable. SciPy (and plotting in general) is not a hard dependency.

Reply:
I'd probably just move the SciPy import into the function(s).

Reply:
Ah yeah, like how the `plot` module does. Will do.